dotnet/src/Planners/Planners.Core/Extensions/PromptTemplateConfigExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the namespace of IKernel
+using System.Linq;
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.SemanticFunctions;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Extension methods for PromptTemplateConfig
+/// </summary>
+internal static class PromptTemplateConfigExtensions
+{
+    /// <summary>
+    /// Set the max_tokens request setting to be used by OpenAI models
+    /// </summary>
+    /// <param name="config">PromptTemplateConfig instance</param>
+    /// <param name="maxTokens">Value of max tokens to set</param>
+    internal static void SetMaxTokens(this PromptTemplateConfig config, int maxTokens)
+    {
+        config.Models ??= new();
```

Comment:
Models being null looks like a bug, shouldn't it be empty in the initialization?